.github/CODEOWNERS

```diff
@@ -1191,3 +1191,6 @@ packages/kbn-yarn-lock-validator @elastic/kibana-operations
 
 # Design (at the bottom for specificity of SASS files)
 **/*.scss  @elastic/kibana-design
+
+# Changes to translation files should not ping code reviewers
+!x-pack/plugins/translations/translations @elastic/kibana-localization
```

Comment:
I don't think this is valid syntax (highlighted at https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax)

I believe we need to set an empty code owners line like 

```# Changes to translation files should not ping code reviewers
x-pack/plugins/translations/translations 
```